packages/runtime-core/src/vgo_runtime/router_contract_presentation.py

```
@@ -5,11 +5,43 @@
 from .router_contract_support import RepoContext, read_skill_descriptor
 
 
+# UI string constants for confirm UI rendering
+CONFIRM_UI_BATCH_PROMPT = "请尽量一次性回答下面问题；能合并回答的内容可以放在同一条消息里："
```

Ruff (0.15.10)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)